data-types/collections/hashes.rb

```diff
@@ -1,3 +1,17 @@
+#hashes.rb contains a series of methods that need to be completed by you.
+
+#For each method,
+#1. Read the comments and attempt to add or modify the code to complete.
```

Comment:
some of these instructions, while super on-point, are also a bit verbose to stick straight into the test file. Do you think they could be more appropriate for a `README.md` in a place "close" to this file?

Reply:
good call, since all the files in this folder seem to be laid out the same, the written directions could probably just be in one file.